### PR TITLE
s3gw container for docker

### PR DESCRIPTION
**Pull request by giubacc**
- User can now choose to build the s3gw container with docker also.
  Regardless of what is employed - podman or docker engine - to build it, produced image will be OCI compliant. 

Fixes: aquarist-labs/s3gw#27
Signed-off-by: Giuseppe Baccini <giuseppe.baccini@suse.com>

### CODE REVIEW

File: build/README.md [hunk 1]
```diff
@@ -54,6 +58,14 @@ $ cd ~/git/s3gw-core/build
 $ ./build-container.sh
 ```
 
+By default, this will build an `s3gw` image using podman.
+In order to build an `s3gw` image with docker, you can run:
```
Comment (votdev) [inline build/README.md:62]:
```suggestion
In order to build an `s3gw` image with Docker, you can run:
```

File: build/README.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ Make sure you've installed the following applications:
 - podman
 - buildah
 
+Optionally, if you prefer building an `s3gw` container image with docker you will need:
```
Comment (votdev) [inline build/README.md:13]:
```suggestion
Optionally, if you prefer building an `s3gw` container image with Docker you will need:
```

File: build/README.md [hunk 3]
```diff
@@ -67,3 +79,9 @@ Finally, you can run the `s3gw` container with the following command:
 ```
 $ podman run --replace --name=s3gw -it -p 7480:7480 localhost/s3gw
 ```
+
+or, when using docker:
```
Comment (votdev) [inline build/README.md:83]:
```suggestion
or, when using Docker:
```

File: build/build-container.sh [hunk 4]
```diff
@@ -46,6 +51,11 @@ EOF
     buildah push ${registry_args} localhost/${IMAGE_NAME} \
       ${registry}/${IMAGE_NAME}
   fi
+      ;;
+    docker)
+    docker build -t ${IMAGE_NAME} -f ./Dockerfile.build-container ${CEPH_DIR}/build
```
Comment (votdev) [inline build/build-container.sh:56]: Indention is not correct here.

**Review by votdev — CHANGES_REQUESTED** [review]

**Comment by giubacc** [comment]
There is an issue with package `libicu70`, it seems no more available on repository; I'm investigating.

**Comment by votdev** [comment]
> There is an issue with package `libicu70`, it seems no more available on repository; I'm investigating.

Indeed 🤬

**Comment by giubacc** [comment]
@votdev I have updated `libicu70 -> libicu71` for podman also, I ran the container and it works

**Pull request merged**